views.sql

```diff
@@ -0,0 +1,21 @@
+CREATE OR REPLACE VIEW daily_booking_alterations as
+SELECT
+    s.C_ID,
+    st.STAFF_ID,
+    se.STATUS,
+    se.EVENT_DATE,
+    se.EVENT_START,
+    se.EVENT_END
+FROM EDH_WMS_STAGING.NGT_SITE_EVENTS se
+JOIN EDH_WMS_STAGING.NGT_SITE s
+ON se.SITE_ID = s.SITE_ID
+JOIN EDH_WMS_STAGING.NGT_STAFF st
+ON se.STAFF_ID = st.STAFF_ID
```

Comment:
would typically indent lines 11 and 13 if not keeping on the same line as the table name